solutions/swb-reference/src/proxyHandlerLambda.ts

```diff
@@ -190,14 +187,11 @@ export async function handler(event: any) {
     error.isBase64Encoded = false;
     error.headers = { 'Content-Type': 'application/json' };
 
-    console.error(`Application error encountered: ${JSON.stringify(error)}`);
+    console.error('Application error encountered', error);
```

Comment:
Are you sure the error (non-string type) object can be read in CW logs? Or did you mean to write `error.message` here?

Reply:
Yes, we use the same syntax in other places of the code base. [Here](https://github.com/aws-solutions/research-service-workbench-on-aws/pull/1071/files#diff-f1abf758c3206eeb07deec5194b081e75de50255015a900efcd2f07a7a038448) for example.